schema/voters.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = {
+	getVoters: {
+		id: 'delegates.getVoters',
+		type: 'object',
+		properties: {
+			address: {
+				type: 'string',
+				format: 'address',
+				minLength: 21,
```

Comment:
minLength 2

Reply:
minLength needs to support the full range of addresses

Reply:
my bad, thanks!